packages/manager/modules/emailpro/src/dashboard/emailproServices.module.js

```diff
@@ -12,6 +13,7 @@ const moduleName = 'Module.emailpro.services';
 
 angular
   .module(moduleName, [ovhManagerCore])
+  .service('messaging', messaging)
```

Comment:
Why this service is not currently used ?

Reply:
It is used, in EmailPro KdimAutoConfigurator